### PR TITLE
Converts butterflies to basic mobs and adds them to cytology

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

Converts butterflies into basic mobs.
Also a little list organisation.

<details>
<summary>Yep, those are some butterflies alright</summary>

image

</details>

## Changelog
:cl:
refactor: Converted butterflies to the basic mob system
add: Butterflies can now be grown in cytology
/:cl:

### CODE REVIEW

**Comment by Mothblocks** [comment]
You need an UpdatePaths script to convert the simples to basics, please look at existing ones to know how to do it

File: code/modules/events/wizard/petsplosion.dm [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-	/mob/living/basic/lizard,
-	/mob/living/simple_animal/sloth,
-	/mob/living/simple_animal/hostile/retaliate/goat,
-	/mob/living/simple_animal/chicken,
 	/mob/living/basic/bat,
-	/mob/living/simple_animal/butterfly,
-	/mob/living/simple_animal/hostile/retaliate/snake,
-	/mob/living/simple_animal/hostile/retaliate/goose/vomit,
+	/mob/living/basic/butterfly,
+	/mob/living/basic/lizard,
```
Comment (Mothblocks) [inline code/modules/events/wizard/petsplosion.dm:5]: Why is lizard here? Looks like you made this ABC order except for this

File: code/modules/mob/living/basic/vermin/butterfly.dm [hunk 2]
```diff
@@ -1,38 +1,43 @@
...
+	ai_controller = /datum/ai_controller/basic_controller/butterfly
+
+/mob/living/basic/butterfly/Initialize(mapload)
 	. = ..()
 	AddElement(/datum/element/simple_flying)
+
 	var/newcolor = rgb(rand(0, 255), rand(0, 255), rand(0, 255))
 	add_atom_colour(newcolor, FIXED_COLOUR_PRIORITY)
 	ADD_TRAIT(src, TRAIT_VENTCRAWLER_ALWAYS, INNATE_TRAIT)
+	AddElement(/datum/element/swabable, CELL_LINE_TABLE_BUTTERFLY, CELL_VIRUS_TABLE_GENERIC_MOB, 1, 5)
```
Comment (Mothblocks) [inline code/modules/mob/living/basic/vermin/butterfly.dm:36]: Named parameters for 1 and 5

File: code/modules/events/scrubber_clog.dm [hunk 3]
```diff
@@ -65,10 +65,10 @@
 
 /datum/round_event/scrubber_clog/proc/get_mob()
 	var/static/list/mob_list = list(
+				/mob/living/basic/butterfly,
 				/mob/living/basic/cockroach,
 				/mob/living/basic/giant_spider/maintenance,
 				/mob/living/basic/mouse,
```
Comment (san7890) [inline code/modules/events/scrubber_clog.dm:71]: you're the last person who's gonna touch this list so

```suggestion
		/mob/living/basic/butterfly,
		/mob/living/basic/cockroach,
		/mob/living/basic/giant_spider/maintenance,
		/mob/living/basic/mouse,
```

**Review by san7890 — COMMENTED** [review]
thanks for alphabetizing everything, that's well appreciated. i'm going to assume direct control on that list multilining style nit

**Pull request merged**